src/Store/rocketReducer.jsx

```diff
@@ -0,0 +1,64 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import axios from "axios";
+//Async
+const rocketThunk = createAsyncThunk("rocket", async () => {
+  try {
+    try {
+      const respons = await axios.get("https://api.spacexdata.com/v3/rockets");
+      return respons.data;
+    } catch (error) {
+      return error;
+    }
+  } catch (error) {}
+});
+const initialState = {
```

Comment:
Here you use to try-catch block, I would suggest removing one of them. Because it makes your code unclear.